doc/source/guide/inputtext-flags.rst

```diff
@@ -0,0 +1,28 @@
+.. _guide-inputtext-flags:
+
+Using input text flags
+==================
```

Comment:
Make the underline same width as the title. Otherwise Sphinx will generate warnings.